src/doc/octree_map.cpp

```diff
@@ -118,6 +118,27 @@ color_t OctreeNode::hextetToBranchColor(int hextet, int level)
 //////////////////////////////////////////////////////////////////////
 // OctreeMap
 
+// This constructor initializes 'm_maskIndex' and
+// 'm_includeMaskColorInPalette' according to sprite's pixel format and
+// background existence.
+// This variables are needed before makePalette() and mapColor()
```

Comment:
Sorry I missed this in my previous review
```suggestion
// These variables are needed before makePalette() and mapColor()
```

Reply:
Done